plugins/Morpheus/stylesheets/less/_variables.less

```diff
@@ -17,6 +17,8 @@
 @theme-color-background-highContrast:  #202020;
 @theme-color-base-series:              #ee3024;
 
+@theme-color-border: @color-silver-l80;
```

Comment:
Maybe we could comment the actual color value next to those variables so should be easy to find then? Maybe it's even possible to show in the style guide page? Although we probably would not want to show the colors from `_colors.less` there.

I was really confused by that variable and wouldn't know when to use it.